verticaltabs.jsm

```diff
@@ -287,11 +288,15 @@ VerticalTabs.prototype = {
     toolbar.appendChild(pin_button);
     leftbox.insertBefore(toolbar, leftbox.firstChild);
 
+    // change the text in the tab context box
+    let close_next_tabs_message = document.getElementById('context_closeTabsToTheEnd');
+    close_next_tabs_message.setAttribute('label', 'Close Tabs Below');
```

Comment:
So, on the one hand, I worry a little about this not getting localized.  On the other hand, we're only releasing in en_US for the first while.  So I guess it's okay for now, but should probably be fixed for real later…